posts/Fabio/2018-04-16-hibernate-ogm-5-4-Alpha1-released.adoc

```diff
@@ -0,0 +1,68 @@
+= "Hibernate OGM 5.4.0.Alpha1 is out"
+Fabio Massimo Ercoli
+:awestruct-tags: [ "Hibernate OGM", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+We're happy to announce the release of:
+http://hibernate.org/ogm/releases/5.4/#get-it[Hibernate OGM 5.4.0.Alpha1]
+
+Hibernate OGM is now compatible with Hibernate ORM 5.3 and JPA 2.2!
+
+== Component upgrades
+
+ * Hibernate ORM to 5.3.0.CR1
+ * Hibernate Search to 5.10.0.Beta2
+ * WildFly to 12.0.0
+ * Infinispan to 9.2.0.Final
+ * MongoDB to 3.6.3
+ * Neo4J to 3.3.5
+
+== Module Distribution
+
+ * WildFly modules are now available as feature packs
+
+== Infinispan
+
+ * Caches created by the dialect are now transactional by default
+ * Caches created by the dialect can be configured using a cache configuration
+
+== MongoDB
+
+ * Stored procedures with positional parameters are now supported
+ * New MongoDB CLI operations supported as native queries
```

Comment:
I think we can be more specific and change it to  "dropping a colleciton" in mongo db with a native query.
Maybe with thanks to the guy who did it.